### PR TITLE
feat: Payment & Shipping Methods added

### PR DESCRIPTION
**Pull request by pronob1010**
Payment & Shipping Methods added

image

## Related PR : https://github.com/thrivedesk/app-frontend/pull/912

### CODE REVIEW

File: src/Plugins/WooCommerce.php [hunk 1]
```diff
@@ -340,11 +342,11 @@ public function get_order_items( $order ): array {
 		}
 
 		foreach ( $items as $item ) {
-
-			$product = wc_get_product( $item["product_id"] );
+			
+			$product_id = $item->get_variation_id() ? $item->get_variation_id() : $item->get_product_id();
```
Comment (Rijoanul-Shanto) [inline src/Plugins/WooCommerce.php:346]:
```php
$product_id = $item->get_variation_id() ?? $item->get_product_id();

```

File: src/Services/PortalService.php [hunk 2]
```diff
@@ -24,6 +24,7 @@ class PortalService {
         'starter_july_23',
         'starter_annual_july_23',
 		'business_ltd',
+		'business-ltd',
```
Comment (Rijoanul-Shanto) [inline src/Services/PortalService.php:27]: We need to make this smarter. We shouldn't rely on adding plans directly here. Things should be verified via the ThriveDesk API.

**Review by Rijoanul-Shanto — CHANGES_REQUESTED** [review]
Almost LGTM 😃

File: src/Services/PortalService.php (hunk 2 above)
Comment (pronob1010) [reply src/Services/PortalService.php:27]: okay bhai

**Pull request merged**